src/data/pokemon/species_info/gen_9_families.h

```diff
@@ -3215,12 +3215,12 @@ const struct SpeciesInfo gSpeciesInfoGen9[] =
         .iconSprite = gMonIcon_Scovillain,
         .iconPalIndex = 1,
         FOOTPRINT(Scovillain)
-        OVERWORLD(
+        OVERWORLD_SET_ANIM(
             sPicTable_Scovillain,
```

Comment:
East facing sprite has red filling in the space between body and head where it should be transparent